schemas/context/destination-namespace.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/destination-namespace",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Destinations Namespace Mapping",
+  "type": "object",
+  "meta:extensible": false,
+  "meta:abstract": false,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/data/record"
+  ],
+  "description": "Destinations Namespace Mapping Schema.",
+  "definitions": {
+    "destination-namespace": {
+      "properties": {
+        "xdm:destinationID": {
+          "title": "Destination ID",
+          "type": "string",
+          "description": "Unique Identifier for destination flow"
+        },
+        "xdm:namespaceCode": {
+          "title": "Namespace Code",
+          "type": "string",
+          "description": "This code is a human readable identifier for the namespace and can be used to request the technical namespace id which is used for identity graph processing."
+        },
+        "xdm:createTime": {
+          "title": "Create Time",
+          "type": "string",
```

Comment:
there is already a standard `xmp:createDate` value that you should be using instead.   And it is defined as a `date-time` in ISO 8601 (which you also need to be using)